sass/layout/_main-card.scss

```diff
@@ -0,0 +1,116 @@
+.featured_article {
+  display: flex;
```

Comment:
These line of code are being repeated, check if there is no @mixin for that. Or create a new one.